docs/building.md

```diff
@@ -20,5 +20,5 @@ dist:
 processors:
   - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/processor/filterprocessor v0.50.0"
   - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricstransformprocessor v0.50.0"
-  - gomod: "github.com/honeycombio/opentelemetry-collector-configs/timestampprocessor v0.4.0"
+  - gomod: "github.com/honeycombio/opentelemetry-collector-configs/timestampprocessor v0.1.0"
```

Comment:
Latest version is `v1.4.0`

```suggestion
  - gomod: "github.com/honeycombio/opentelemetry-collector-configs/timestampprocessor v1.4.0"
```

Reply:
Bah, I typo'd the fix for the typo. Of course.